webapp/sources/ldap-inventory/inventory-api/src/main/scala/com/normation/inventory/services/provisioning/InventoryParser.scala

```diff
@@ -92,9 +92,10 @@ object XmlInventoryParser {
     try {
       val f = SAXParserFactory.newInstance()
       f.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true)
-      f.setNamespaceAware(false)
       f.setXIncludeAware(false)
       f.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true)
+      // this one is from Scala XML implementation of new SAX parseur, so keeping it to avoid surprises
+      f.setNamespaceAware(false)
```

Comment:
it seems to be the default value anyway

Reply:
I don't want to have it changed by a new JVM version or whatever

Reply:
sure, but it also means it is very likely not a problem :+1: